discord/monetization.py

```diff
@@ -101,12 +107,80 @@ def url(self) -> str:
         """:class:`str`: Returns the URL for the SKU."""
         return f"https://discord.com/application-directory/{self.application_id}/store/{self.id}"
 
+    def fetch_subscriptions(
+        self,
+        user: Snowflake,  # user is required because this is a bot, we are not using oauth2
+        *,
+        before: SnowflakeTime | None = None,
+        after: SnowflakeTime | None = None,
+        limit: int | None = 100,
+    ) -> SubscriptionIterator:
+        """Returns an :class:`.AsyncIterator` that enables fetching the SKU's subscriptions.
+
+        .. versionadded:: 2.7
+
+        Parameters
+        ----------
+        user: :class:`.abc.Snowflake`
+            The user to retrieve subscriptions for.
```

Comment:
```suggestion
            The user for which to retrieve subscriptions.
```
Sentences shouldn't end with a preposition